spec/javascripts/app/utils/ms-formatter_spec.js

```diff
@@ -3,11 +3,11 @@ import msFormatter from '../../../../app/javascript/app/utils/ms-formatter';
 describe('#msFormatter', () => {
   it('formats milliseconds as X:XX', () => {
```

Comment:
let's update the comment to match?
```suggestion
  it('formats milliseconds as XX:XX:XX', () => {
```